live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandlerTest.java

```diff
@@ -80,6 +84,8 @@ class IngestMetadataCommandHandlerTest
     ArgumentCaptor<UpdateNodeMetadataEvent> updateNodeMetadataEventCaptor;
     @Mock
     EventPublisher eventPublisher;
+    @Spy
+    List<CustomPropertyResolver<?>> customPropertyResolvers = Collections.emptyList();
```

Comment:
Is this `@Spy` used anywhere?

Reply:
it is required because we have to inject a real collection as we are iterating over it